client/src/components/Project/Scrums/index.jsx

```diff
@@ -0,0 +1,264 @@
+                <Editable
+                  html={value}
+                  onChange={event => this.handleEditTask(event, record, 'est_time')}
+                  tagName="span"
+                  className="tasks__cell estimate_time"
+                />
+              );
+            },
+          },
+          {
+            title: 'Remaining Time (hr)',
+            dataIndex: 'remaining_time',
+            render: (value, record) => {
+              return (
+                <Editable
+                  html={value}
+                  onChange={event => this.handleEditTask(event, record, 'remaining_time')}
+                  tagName="span"
+                  className="tasks__cell remaining_time"
+                />
+              );
+            },
+          },
+          {
+            title: 'Status',
+            dataIndex: 'status',
+            render: (value, record) => {
+              return (
+                <StatusSelect onChange={event => this.handleEditTask(event, record, 'status')} defaultValue={record.status} />
+              );
+            },
+            filters: Filter(this.state.tasks).statusFilters,
+            onFilter: (value, record) => record['status'] === value
+          },
+          {
+            title: 'Assignee',
+            dataIndex: 'assignee',
+            render: (value, record) => {
+              return (
+                <Editable
+                  html={value}
+                  onChange={event => this.handleEditTask(event, record, 'assignee')}
+                  tagName="span"
+                  className="tasks__cell"
+                />
+              );
+            },
+            filters: Filter(this.state.tasks).assigneesFilters,
+            onFilter: (value, record) => record['assignee'] === value,
+            sorter: (a, b) => Sort(a, b, 'assignee'),
+          },
+          {
+            title: 'Ticket',
+            dataIndex: 'ticket',
+            render: (value, record) => {
+              return (
+                <Editable
+                  html={value}
+                  onChange={event => this.handleEditTask(event, record, 'ticket')}
+                  tagName="span"
+                  className="tasks__cell"
+                />
+              );
+            },
+          },
+          {
+
+            render: record => {
+              return (
+                  <Icon type="delete" onClick={this.handleDeleteTask}/>
+              );
+            },
+          }
+          ];
+          render(){
+            const columns = this.columns;
+            if (this.state.saved) {
+              setTimeout(() => {
+                this.setState({ saved: false})
+              }, 4000);
+            }
+          return(
+          <React.Fragment>
+            <section className='Scrum__page--container'>
+              {/* <h2> {this.state.scrumName} </h2> */}
+              <div className='Scrum__header'>
+                <h2 className='Scrum__name'> Scrum Name </h2>
```

Comment:
I don't think we need Scrum Name here, it's already on the tab
Also, don't forget to remove commented code